cmd/gsw_service.go

```diff
@@ -112,12 +112,16 @@ func readConfig() *viper.Viper {
 	config.AddConfigPath("data/config/")
 	err := config.ReadInConfig()
 	if err != nil {
-		panic(fmt.Errorf("Error reading GSW config: %w", err))
+		logger.Error(fmt.Sprint("Error reading GSW config: %w", err))
```

Comment:
Keep the panic. This exits the entire program. If we can't read the config file at all, we cant run GSW anyways.